core/src/main/java/org/sonarsource/sonarlint/core/websocket/SonarCloudWebSocket.java

```diff
@@ -100,8 +100,10 @@ private void handleRawMessage(String message, Consumer<ServerEvent> serverEventC
   }
 
   private static Optional<? extends ServerEvent> parse(WebSocketEvent event) {
-    var eventType = event.eventType;
-    if (!parsersByType.containsKey(eventType)) {
+    var eventType = event.event;
+    if (eventType == null){
+      return Optional.empty();
```

Comment:
I separated these two checks because if the received message is not an event we want to simply ignore and not log an error.